taqueria-vscode-extension-web-ui/src/data-editors/PairEditor.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { DataEditorNode } from './DataEditorNode';
+import { getFriendlyDataType } from './MichelineEditor';
+
+export const PairEditor = (
+	{ dataType, value, onChange }: { dataType: any; value: any; onChange: (value: any) => void },
```

Comment:
Is it possible to avoid the `any` type here?

Reply:
Later in the coding process I have declared a `MichelineValue` type. I can try to use it in here.